### ISSUE TITLE
kaiju-makedb mar database

### ISSUE THREAD

**Issue by typlam** [bioinformatics-centre/kaiju#232]
Hi,
I am trying to set up the reference db with Mar database via the "kaiju-makedb -s mar" command, and got the following error message:

```
\033[0;32mConverting MarRef data to Kaiju format\033[0m
  File "/scratch/tl2y12/kaiju/bin/kaiju-convertMAR.py", line 98
    db_map = {**db_map, **tmp}
                        ^
SyntaxError: invalid syntax
```
I didn't change anything on the python script 'kaiju-convertMAR.py', and I compared with the one on github here and saw no difference between. 

Any recommendations?  Thanks!

**Comment by typlam** [comment]
something shifted - that ^ should be under the first double asterisks **

**Comment by pmenzel** [comment]
Hi,

I suspect that it is an issue with python version.
Do you have Python3 installed? Check by just running `python` on the command line.

**Comment by typlam** [comment]
You're right. I forgot to check which version was switched on. Many thanks for your quick response!
